server/pbench/bin/pbench-inotify

```diff
@@ -219,6 +224,10 @@ def main():
     # initialization period: any pbench-move-results will have to wait
     # for this to finish.
```

Comment:
What does this mean? `pbench-move-results` are run asynchronously by agents all over the place. They might very well be depositing tarballs into this directory even as this script initializes. But IIRC, pbench-move-results takes a lock on the directory, so we can take the same lock exclusively: that way we wait for them to finish, before starting our initialization. But then we have to process any tarballs that we find in this directory, something which I don't see in the code currently.